src/main/java/com/xebia/incubator/xebium/AddMissingSeleneseCommands.java

```diff
@@ -0,0 +1,64 @@
+package com.xebia.incubator.xebium;
+
+import com.thoughtworks.selenium.webdriven.SeleneseCommand;
+import com.thoughtworks.selenium.webdriven.WebDriverCommandProcessor;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.interactions.ContextClickAction;
+import org.openqa.selenium.interactions.HasInputDevices;
+import org.openqa.selenium.interactions.Mouse;
+import org.openqa.selenium.internal.Locatable;
+
+public class AddMissingSeleneseCommands {
```

Comment:
Hmm sorry this is not quite what I meant: in this class you add 2 anonymous subclasses of `SeleneseCommand<Void>`. What I meant was that it might be nice to make those named top-level classes.